examples/forward/plot_read_bem_surfaces.py

```diff
@@ -22,11 +22,23 @@
 subjects_dir = op.join(data_path, 'subjects')
 
 ###############################################################################
-# Here we use :func:`mne.viz.plot_trans` with ``trans=None`` to plot only the
-# surfaces without any transformations. For plotting transformation, see
+# Here we use :func:`mne.viz.plot_alignment` with ``trans=None`` to plot only
+# the surfaces without any transformations. For plotting transformation, see
 # :ref:`tut_forward`.
 
-mne.viz.plot_trans(raw.info, trans=None, subject='sample',
-                   subjects_dir=subjects_dir, meg_sensors=[], eeg_sensors=[],
-                   head='outer_skin', skull=['inner_skull', 'outer_skull'])
+mne.viz.plot_alignment(raw.info, trans=None, subject='sample',
+                       subjects_dir=subjects_dir, meg_sensors=[],
+                       eeg_sensors=[], head='outer_skin',
+                       skull=['inner_skull', 'outer_skull'])
```

Comment:
can we take this opportunity to cleanup the API for other surfaces. So remove head and skull params and do

plot_alignment(..., surfaces=['outer_skin', 'inner_skull', 'outer_skull'])

with surfaces defaulting to the head surface?